### PR TITLE
Update ownerless-groups-teams.md

### PR DESCRIPTION
**Pull request by msbemba**
Mentioned that Global administrator is required to create a policy for ownerless groups

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8543

File: microsoft-365/admin/create-groups/ownerless-groups-teams.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ description: "Learn how to automatically invite members to become owners in an o
 
 A team in Microsoft Teams or a Microsoft 365 group can become ownerless if an owner's account is deleted or disabled in Microsoft 365. Groups and teams require an owner to add or remove members and change group settings.
 
-You can create a policy what automatically asks the most active members or an ownerless group or team if they'll accept ownership. When a member accepts the invitation to become an owner, the action is logged in the compliance portal audit log. Guests are never invited to be owners.
+A Global administrator create a policy what automatically asks the most active members or an ownerless group or team if they'll accept ownership. When a member accepts the invitation to become an owner, the action is logged in the compliance portal audit log. Guests are never invited to be owners.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/ownerless-groups-teams.md]:
```suggestion
A Global administrator can create a policy that automatically asks the most active members of an ownerless group or team if they'll accept ownership. When a member accepts the invitation to become an owner, the action is logged in the compliance portal audit log. Guests are never invited to be owners.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**